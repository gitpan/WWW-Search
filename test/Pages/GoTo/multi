http://www.goto.com/d/search/p/befree/?Keywords=goldeen
/d/search/;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?Keywords=goldeen&view=2+39+1
