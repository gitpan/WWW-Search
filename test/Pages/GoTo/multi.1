
  <html>
<head>
 <gotostate name="merope.back">
 <gotostate userAdultStatus="expired">
 <gotostate bannerAdultStatus="cleanBanner">
 <gotostate queryAdultStatus="cleanQuery">
 <gotostate searchRatings="">

 <title>GoTo.com Search Results:
  goldeen</title>
 

<script language="JavaScript">
//<!--
function launchGoToRemote() {
  window.open("/d/remote-control/remote.jhtml","gotoremote",
  "width=150,height=150,location=0,menubar=0,resizable=0,scrollbars=0,status=0,titlebar=1,toolbar=0");
}
function infoWindow() { window.open('/d/search/shared/adult/moreinfo.jhtml','',
  'toolbar=0,location=0,directories=0,status=0,menubar=0,scrollbars=yes,resizable=0,width=482,height=380'); }
//-->
</script>
</head>

<body bgcolor="#FFFFFF" text="#000000">

 <center>

  <table border=0 cellpadding=0 cellspacing=0 width=610 bgcolor="#FFCC00"><form method=GET action="/d/search/;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ" name=Search target="_top"><input type=hidden name=type value=topbar><tr><td rowspan=3 width=132><a href="http://www.goto.com/" target="_top"><img src="/images/shared/lil.gif" border=0 width=132 height=105 hspace=0 vspace=0 alt="GoTo.com"></a></td><td colspan=2 align=right valign=top bgcolor="#ffcc00"><iframe WIDTH=468 HEIGHT=60 NORESIZE SCROLLING=No FRAMEBORDER=0 MARGINHEIGHT=0 MARGINWIDTH=0 SRC="http://adforce.imgis.com/?adiframe|2.0|2|80148|1|1|KEY=CLEAN_goldeen;"><a href="http://adforce.imgis.com/?adlink|2.0|2|80148|1|1|KEY=CLEAN_goldeen;" target="_top"><img src="http://adforce.imgis.com/?adserv|2.0|2|80148|1|1|KEY=CLEAN_goldeen;" border=0 width=468 height=60 alt="GoTo.com ad"></a></iframe></td></tr><tr><td rowspan=2>&nbsp;&nbsp;</td><td><table border=0 cellpadding=3 cellspacing=0 bgcolor="#FFCC00"><tr valign=middle><td><input type=text value="goldeen" name=Keywords size=30></td><td><input type=image src="/images/shared/ar2.gif" alt="GoTo It!" border=0></td></tr></table></td></tr><tr valign=top><td><img src="/images/shared/st2.gif" width=236 height=12 border=0 hspace=0 vspace=0 alt="Simply type what you're looking for and GoTo It!"></td> </form></tr></table>
<gotostate pop=Banner1Control>



<br clear=left>
<table border=0 width=610  cellspacing=0 cellpadding=1>
 <tr><td align=left valign=middle>











&nbsp;</td><td align=right valign=middle>
<font face="helvetica, arial" size=2>
  <a href="/d/about/advertisers/test.jhtml;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ" target="_top">List Your Site For $25</a> | <a href=/d/about/howto/ht_bookmark.jhtml;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ
  target=_top><nobr>Bookmark GoTo</nobr></a></font>
  <font face="helvetica, arial" size=2> | 
  <a href="/d/popular/;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ" target="_top">Try Our Popular Searches!</a></td> </tr>
 </table>
<br clear=left>


<a name=41><table border=0 width=610 cellspacing=5 cellpadding=1><tr><td align=right width=20><font face="Helvetica, Arial" size=-1>41.</font></td><td width=590><font face="Helvetica, Arial"><b><a href=/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEiyNLkwpzSkpSi%2FL10otigfkppcmpuSV5QMVadk5%2BPmZOkE1KzizRgZWpwyAWIBiaJarmJ%2Bfl2hXnpqTnpqQh5mgc5uBo6ulsbmhsbGxuYWgOaBtxGUyIAC8O9A%3D%3D target=_top>Bruce Beasley</a></b></font></td></tr><tr valign=top><td align=right>&nbsp;</td><td><font face="Helvetica, Arial" size=-1>Bruce Beasley &quot;Bruce Beasley's works are space - enveloping and space - defining. His engagement with spatial problems shows association with Eduardo Chillida's work, with whom he is a friend. Beasley makes space something that can be  vigorously..<br><em>http://www.sculpture.org/</em>&nbsp;</font></td></tr></table></a>
<a name=42><table border=0 width=610 cellspacing=5 cellpadding=1><tr><td align=right width=20><font face="Helvetica, Arial" size=-1>42.</font></td><td width=590><font face="Helvetica, Arial"><b><a href=/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEMyNK83NSMnJLi4sSGkASQanmOblJqTmpdkBiempuiDHKTiyvz8gCLgOZy7FIxDktLzGvPzbQPz0lJzU14Q8DNlc3Y0dHS2Nzc2NDY0MLiHNgswDK9WvQLQ%3D%3D target=_top>CD: Gameboy Games: Pokemon Red</a></b></font></td></tr><tr valign=top><td align=right>&nbsp;</td><td><font face="Helvetica, Arial" size=-1>Pokemon Red Version I think this game has some really cool music. This is a wonderful RPG. The variety of monsters to catch and train makes this game addicting. EGM rated 8.5/10 Game Informer rated 9/10 Cute monsters are the highlight. The rest  of.<br><em>http://www.intellistar.net/</em>&nbsp;</font></td></tr></table></a>
<a name=43><table border=0 width=610 cellspacing=5 cellpadding=1><tr><td align=right width=20><font face="Helvetica, Arial" size=-1>43.</font></td><td width=590><font face="Helvetica, Arial"><b><a href=/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlElEiUV5mfm6umn5BRn5JXrJqboZ%2BcUlaW0g2UCi5JLM1JJiosKQ%2FMywYhARTHvVjBOS8vMK89Ntc%2FNSUvNT7hBwMnVztjQ0dDY3Nza0NjQwOIemCQAMXaVXLA%3D%3D target=_top>Critters Workshop Bios</a></b></font></td></tr><tr valign=top><td align=right>&nbsp;</td><td><font face="Helvetica, Arial" size=-1>Critters Workshop Member Biographies The members who have agreed to have their bios listed are presented below. Note that we have writers from a variety of backgrounds (beginners to Hugo-nominees, teenagers to retirees, from various continents,  etc<br><em>http://brain-of-pooh.tech-soft.com/</em>&nbsp;</font></td></tr></table></a>
<a name=44><table border=0 width=610 cellspacing=5 cellpadding=1><tr><td align=right width=20><font face="Helvetica, Arial" size=-1>44.</font></td><td width=590><font face="Helvetica, Arial"><b><a href=/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEkyMyizJLSwv1U4vygdKlQqnFxUUaQFwkYQJlJhfm9YMsQzExS1HMTs%2FLtCvPTUnPTUlDzco4zcDR1dLY3NHY2NjcwtAc0B7iMpg%2FACtonQ%3D%3D target=_top>Critters Workshop Bios</a></b></font></td></tr><tr valign=top><td align=right>&nbsp;</td><td><font face="Helvetica, Arial" size=-1>Critters Workshop Member Biographies The members who have agreed to have their bios listed are presented below. Note that we have writers from a variety of backgrounds (beginners to Hugo-nominees, teenagers to retirees, from various continents,  etc<br><em>http://www.critique.org/</em>&nbsp;</font></td></tr></table></a>
<a name=45><table border=0 width=610 cellspacing=5 cellpadding=1><tr><td align=right width=20><font face="Helvetica, Arial" size=-1>45.</font></td><td width=590><font face="Helvetica, Arial"><b><a href=/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlBlEufkFqemJxamZeml5xfmpqSDZIB3zOazUeKwTczMLIAvAL9vf19UP34diqYlyWl5iXn5toX56Sk5qasIeBk2ubsaOjpbG5ubGhsaGFhDmwOcBlGwAKRkG target=_top>http://homepages.infoseek.com/</a></b></font></td></tr><tr valign=top><td align=right>&nbsp;</td><td><font face="Helvetica, Arial" size=-1>&lt;HTML&gt;&lt;HEAD&gt;&lt;TITLE&gt;My POKEMON Page&lt;/TITLE&gt;&lt;/HEAD&gt;&lt;BODY text=000000 link=0000ff vlink=000066 bgcolor=ffffff background=&quot;&quot;&gt; &lt;font face=&quot;&quot; size=&quot;&quot;&gt; &lt;center&gt;&lt;table width=&quot;600&quot; cellpadding=&quot;2&quot; border=0&gt; &lt;TR&gt; &lt;TD width=&quot;300&quot; valign=top&gt;&lt;font face=&quot;&quot;  size<br><em>http://homepages.infoseek.com/</em>&nbsp;</font></td></tr></table></a>
<br><br clear=left>
<table border=0 width=610 cellspacing=0 cellpadding=1>
 <tr><td align=left valign=middle>











&nbsp;</td><td align=right valign=middle>

  
  
 </td> </tr>
 </table>


  <br clear=left>
  <gotostate pop=Banner1Control>
<table border=0 cellpadding=0 cellspacing=0 width=610 bgcolor="#FFCC00"><tr width=610><td rowspan=3 width=132><a href="http://www.goto.com/" target="_top"><img src="/images/shared/lil.gif" border=0 width=132 height=105 hspace=0 vspace=0 alt="GoTo.com"></a></td><td colspan=2 align=right valign=top bgcolor="#ffcc00">
<a href="/d/redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?url=http%3A%2F%2Fwww.goto.com%2Fd%2Faffiliate%2Fp%2Fbefree%2F%3FPromo%3Dbottombar&special=bottombanner" onmouseover="window.status='Become a GoTo affiliate and make money from your site!';return true;" onmouseout="window.status='';"><img src="/images/banners/goto/affiliate_03_99.gif" border=0 width=468 height=60 alt="Make money as a GoTo affiliate."></a>
</td></tr><tr><td rowspan=2>&nbsp;&nbsp;</td><td><form method=GET action="/d/search/;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ" name=Search><input type=hidden name=type value=bottombar><table border=0 cellpadding=3 cellspacing=0 bgcolor="#FFCC00"><tr valign=middle><td><input type=text value="goldeen" name=Keywords size=30></td><td><input type=image src="/images/shared/ar2.gif" width=21 height=21 alt="GoTo It!" border=0></td></tr></table></td></tr><tr valign=top><td><img src="/images/shared/st2.gif" width=236 height=12 border=0 hspace=0 vspace=0 alt="Simply type what you're looking for and GoTo It!"></td></tr></table></form>
<br>

  <table border=0 cellpadding=0 cellspacing=0 width=610>
 <tr>
<td align=center valign=middle width=203><a href="/d/redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?url=http%3A%2F%2Fhome.netscape.com%2Fcomprod%2Fmirror%2F" target="_top"><img src="/images/netscape/netscape_now_static.gif" width=90 height=30 border=0 hspace=0 vspace=0 alt="Netscape"></a></td>
<td align=center valign=middle width=204><a href="http://www.inktomi.com/products/search/partners/goto.html" target="_top"><img src="/images/inktomi/PowByInkMed.gif" width=126 height=40 border=0 alt="Inktomi"></a></td>  
<td align=center valign=middle width=153><a href="/d/about/company/contact.jhtml;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ" onmouseover="window.status='Contact GoTo.com';return true;" onmouseout="window.status='';" target="_top"><img src="/images/about/contact_button.gif" width=90 height=30 border=0 hspace=0 vspace=5 alt="Contact GoTo"></a></td></tr></table>



<BR><BR>
<table border=0 width=610 cellspacing=5 cellpadding=1><tr><td align=center><font face="helvetica, arial" size=2>&copy; 1997-99 GoTo.com, Inc. All rights reserved. Patent pending.<br><a href="/d/about/company/copyright.jhtml;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ" target="_top" onmouseover="window.status='Copyright Act Information';return true;" onmouseout="window.status='';">Copyright Act Information</a><br><a href="/d/about/company/privacypolicy.jhtml;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ" target="_top" onmouseover="window.status='GoTo.com Privacy Policy';return true;" onmouseout="window.status='';">GoTo.com Privacy Policy</a></font>
</td></tr></table>
</center>
</body>
</html>

 

