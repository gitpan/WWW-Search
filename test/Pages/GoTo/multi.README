This set of test-result pages was created on Fri Sep 24 09:51:16 1999
