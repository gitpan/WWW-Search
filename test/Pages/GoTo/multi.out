http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBECyMScuqBc5OTMkGzgcp5omWJicl6JQW5qjlpxlZAempRfqelqouKcQGpmBxaGpiabEAENcW7QNA5LS8xrz820D89JSc1NewPXTgDCwMzc1NDA1MTA0NTE3MTYwOEc8wnXA2NXUzdzSycTV0sjV3A3AleOEAcyCHigZgZADuSzw%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEqyPzElMS8%2FJKQSKB%2BQWp2fm5QHlIFpADNDIAtIXrYFiWpJicn5doV56bk5%2Bakoeaj3CBuaurm7GliYm5pZOrqxuRKcRlMBkAKYwb
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEEyNKi%2BqAKkECgekpOfmpKXmp5WAzENWNE5Ly8wrzoW0IUpgnuBsaupm7mlg4m7pZGrpBuYdsAyk1ACVJPg%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBECyOz8nNT8%2FNKQCKBHiA5UAWQ0TEtRDE5Py%2FQrz02Jz81JQ81AOEDc1dXN2NLExNzSydXVzcj3YhlMNMAJN0y
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBE0yPzUzOTMkuLU4pBYoGZIam5wcWlhUWJQKljnnJRCmJhiGlmDGYgpJj2JarmJ%2Bfl2hXnpqTnpqQh5mWc4G5q6mbsaWLibulkauoG5BFxGUwyACt9Cw%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBE8yOL8jJK85KLE3RLMpNAq4A6eRIpyUWZQEZGRkpJcn6QMZyRYlhibjYIYgaZl3JaXmJefm2hfnpKTmpqwh7maa4Gxq4mbuaWTiaulkauEG7BVwGUIgAtwj0%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEyyOTU%2FJyS9NBU4FqpUJABSCiOz83NS8%2FJTGgvzMkDz8ZrINiWrlicn5eoV56bU5%2BakoeakHCBuaurm7GliYm5pZOrq5uRrcQlMAAAfigxSo%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEEyPS83NTMtJSi4pBYoHF2cJAsMA%2FIDU7PzfID7EGwYwh%2BliWJajmJ%2Bfl2hXnpqTnpqQh5mTc4G5q6mbsaWLibulkauoG5A5xGUzOACphtA%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBE0yOigYkFqekgxUDF3LEgVJORk5qfGJqTl5jkB%2BQUp2f1gixDVLFOS8vMK89NtM%2FPSUnNTbhD3MnVwNjVxM3c0snE1dLI1eINmBwAMgn9RCs%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBESyNSMvIL8rNKQSKB%2BQWp2eVAFkCm0MzsdLxGsOmIBobJanmJefm2hfnpKTmpqQh5mNe4Gxq6mbuaWDibulkaukG5AhwGU%2FoAKWok
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlLlEuampScVFJXqZRfkFIClA1TdB0P%2FPIEdgHx%2FbINHZNdffyKwPYhuhhVqhYnJ%2BXqFeem1OfmpKHmpJwgbmrq5uxpYmJuaWTq6ubkbXEJTAAAFPTCgp
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEcyOLUyqKk4sTUjLzi1MaQeKAYih%2BQH5AInYQ1EWIRobJanmJefm2hfnpKTmpqQh5mYe4Gxq6mbuaWDibulkaukG5BNwGUyQALWut
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEcyPzEoFyikpqQUKBRQWJ%2BSVFlsVA5pA2WZFSXGQMmBmDmYBNY1isaJyWl5hXn5ton56Sk5qacIe5lquBsauJm7mlk4mrpZGrxBswRQBl8VORKw%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlLlEuampScVFJXqlRQWpJYkgGUD1nOG8xOLUopKS1AzkryA%2FIDU7sycEuARsHsQTQ%2BS1vMS8%2FNtC%2FPSUnNTUhDzMu1wNjV1M3c0snE1dLI1dINyCTgMp7wAts1g%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlMlEOWmJuXmZZSBeQH5BanZ%2BbpQek5VcWdICMQXQ2S1UMTk%2FL9CvPTYnPzUlDzUS4QNzV1c3Y0sTE3NLJ1dXNyMjiMpgRAAnm50%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEMyOBdYqKiytSS4tz6kHCgfkFqdn5uUB5VlZeaU56fGpmTglxBMgtiGaGyWp5iXn5toX56Sk5qakIeZlXuBsaupm7mlg4m7pZGrpBuQQcBlODACyqeQ%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoaT0UKyzDC%2FBB6mm0UoQsvBwSV%2Bz1hAddJmDZq7grWe%2F1Wt%2FYROzDzDL6YRWZFBdV5sAthnV5g%2BnSTtkJEDxF51JgbnHg%2BlXjyigeKKCRdEKD5IeYLDFfyHzJMfmnFQIQnHPSzFjfM0Myso%2FbdE2aYgIceU55Ihy3%2FZ2blbcbqbdRtY9bpPxkb2ZhG0T8%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlLlEFYmJBanpIMVAdXJuakZJYlpxUX52QR5uH5Dv4OrtGuIxAdDbLUIxOT8v0K89Nic%2FNSUPNRbhA3NXVzdjSxMTc0snV1c3IzOIymB4ACfwxg%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlLlExZUlmXqpJRWJRaVAeXogVfNh51SixCAcIKs9MRjVEay6YqWhclpeYl5%2BbaF%2BekpOamrCHuY1rgbGriZu5pZOJq6WRq4QbsCHAZSmACgVEw%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBESyNMCxOKUnNKQSKBeamZKel5hhChoYGBrqElsS9IfMQDI%2BS1vMS8%2FNtC%2FPSUnNTUhDzMe1wNjV1M3c0snE1dLI1dINyBTgMpvQAnbuQ%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlEBEuU0giSBFea5yamJGRnKQJpBdnZ%2BbmgefBNbZMVDIOS0vMa8%2FNtA%2FPSUnNTXhD3MSVwNjVxM3c0snE1dLI1eIN2AjAMryCvEm
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEMyOK81MzS1JSMlPLQJOB6rnCxUnJepnplmCFoYGlCYF4xCMj5LW8xLz820L89JSc1NSEPMxzXA2NXUzdzSycTV0sjV0g3IEuAynKACjjHA%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBECyPLElNTc7NBKoFqAULhjjJAvze9KBU0KMhMSDVP0gGWhmaWvvGViYbxegae4a5BCkAiZFrEYyPktbzEvPzbQvz0lJzU1IQ8zTNcDY1dTN3NLJxNXSyNXSDchC4DKbwAL%2BNm
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEyyOTU%2FKS80tAUoFqOUJ5mcXZOkCvMxS4ZmgqcidYeMQTI%2BS1vMS8%2FNtC%2FPSUnNTUhDzMc1wNjV1M3c0snE1dLI1dINyBLgMpLQApM50%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEqyPLKovzUnJKQCKB%2BQGp2eVAlkBie2puRnFRYkg2xCsjeLVTxOT8vEK89Nuc%2FNSUPNRzhA3MXVzdjSxMTc0snF1d3I0uICmBAAPMInEp
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlBlEufkFqemJxak5esmVxfnlINFAnMSM%2FKTEyvwgKigbmp6ak5RUUlnkCVelguaCDWbYpmSclpeYV5%2BbaJ%2BekpOamnCHuZ2rgbGriZu5pZOJq6WRq8QbMGEAZX0DWS8%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoajkUKTSDDRhCTT0MuAmJFki03AuZeyXGmggypHYWjPt5aSrMKx5g%2FN38vNY80A5bCwTJuewgfkOS5IHVABKO3kFxKJdprZejRfDWuKowRXaMh38qa4S51k8ZERvj33dctp0%2BQjg2X0Ci6628H6RAvr6U7eYbknJt4EpRp8xv%2BADv4Qow7
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEyyOL81JyixISSkrzCzJTS0GLgHoQUoqLS5MKc0pKUov6QXYhWRglqOYn5%2BXaFeempOempCHmZNzgbmrqZuxpYuJu6WRq6gbkDnEZTGYALAJv
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEcyNz00r1izKaQSKAqempeTmJ6fkQlaTZxPwUojMgtTc4uD0xGLU0yDUyATYGm9KxUsg5LS8xrz820D89JSc1NeEPc0RXA2NXEzdzSycTV0sjV4g3YOsAyhCe8i8%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEkyNz81JyS5PzMvPSSxLzMkDzgHqJUsmlXiCQMVwZmlKWlhKakh8JX2bipJYW5hfHZRSnJhvG6JqDFYZMb1igbJyWl5hXn5ton56Sk5qacIe5rquBsauJm7mlk4mrpZGrxBswpQBlkKzVNQ%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlElEiamZeXmpJYn5mSB5QLXEoWTSrxDIGC4MTSlLS4lNyQ8ErzNxUkuLc1PjUvON4vUMQYpB5jUsVDZOS8vMK89NtM%2FPSUnNTThD3NLVwNjVxM3c0snE1dLI1eINmD4AMmdUpTE%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlElE00D1imSAZl5eaklifmZIHlAHsSIZNCvEMgZLg9NKUtLiU%2FJDQSvM3NSS4tyC%2BMyi1ORDeL0DkGIQuY3L1I0TkvLzCvPTbXPzUlLzU44QNzV1cHY1sTM3NLJxdXRyNbiDphMADIAOuzQ%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEyyOTU9OKUotLE4tSMktLMkGKgHpgUqAhYoY7IK8zFLh%2BaKpyMV4YyF2IxsbJanmJefm2hfnpKTmpqQh5mbe4Gxq6mbuaWDibulkaukG5BZwGU9oAL4s6
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEiyPS0xpAvID8nNLkIGJoXQBkDlbTMUTYOS0vMa8%2FNtA%2FPSUnNTXhD3MGVwNjVxM3c0snE1dLI1eINzD1AGWfFp8k
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlNlERYl5qakZRal6lZUZqQW5RUWJICVA3dypgsps%2FEao4YimxslqeYl5%2BbaF%2BekpOampCHmYx7gbGrqZu5pYOJu6WRq6QbkB3AZT%2FQAp0VU%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEcyMrExIKU9NBi4FqRUKVifkFqdn5uUB5Y45OTqG8YalIPB4kBCjFsUzYOS0vMa8%2FNtA%2FPSUnNTXhD3MqVwNjVxM3c0snE1dLI1eIN2CDAMrnVd8r
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlLlERYl6xQXFyYkFqUX5xSXlIAkg1PPEghycmpD8glTsfoQliObGyWp5iXn5toX56Sk5qakIeZlHuBsaupm7mlg4m7pZGrpBuQPcBlMjACsmhg%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlHlEeqWl%2BWlFuYk5mSCVQLWCoez83NQ8%2FAsoE6q6YoWxclpeYl5%2BbaF%2BekpOamrCHuY1rgbGriZu5pZOJq6WRq4QbsCHAZT%2BACmONg%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBE0yNzE8tT8zNS8oqLykGiQTmJJdlFGaX5GeleQH5BanZ%2BbogemJVqngxoCTKbYqWxclpeYl5%2BbaF%2BekpOamrCHuZ1rgbGriZu5pZOJq6WRq4QbsGHAZTyAC63%2Fw%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlAVE2fm5qXn5qckleYlFejm5Fg0gcVCmoNK0HJxGkOGIBibJanmJefm2hfnpKTmpqQh5mMe4Gxq6mbuaWDibulkaukG5AdwGU98AKBn4
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEiyNLkwpzSkpSi%2FL10otigfkppcmpuSV5QMVadk5%2BPmZOkE1KzizRgZWpwyAWIBiaJarmJ%2Bfl2hXnpqTnpqQh5mgc5uBo6ulsbmhsbGxuYWgOaBtxGUyIAC8O9A%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEMyNK83NSMnJLi4sSGkASQanmOblJqTmpdkBiempuiDHKTiyvz8gCLgOZy7FIxDktLzGvPzbQPz0lJzU14Q8DNlc3Y0dHS2Nzc2NDY0MLiHNgswDK9WvQLQ%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlElEiUV5mfm6umn5BRn5JXrJqboZ%2BcUlaW0g2UCi5JLM1JJiosKQ%2FMywYhARTHvVjBOS8vMK89Ntc%2FNSUvNT7hBwMnVztjQ0dDY3Nza0NjQwOIemCQAMXaVXLA%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlCBEkyMyizJLSwv1U4vygdKlQqnFxUUaQFwkYQJlJhfm9YMsQzExS1HMTs%2FLtCvPTUnPTUlDzco4zcDR1dLY3NHY2NjcwtAc0B7iMpg%2FACtonQ%3D%3D
http://goto.com/d/search-redirect;$sessionid$S5Z4G5IAA03Z1QFIEFAAPUQ?xargs=00u3hs9yoahSumGpxqaqgGlBlEufkFqemJxamZeml5xfmpqSDZIB3zOazUeKwTczMLIAvAL9vf19UP34diqYlyWl5iXn5toX56Sk5qasIeBk2ubsaOjpbG5ubGhsaGFhDmwOcBlGwAKRkG
